client_test.go

```diff
@@ -57,9 +60,15 @@ var (
 	ctx           = context.TODO()
 	scope         = "test"
 	namePrefix    = "team.service"
-	nullConnector = &devnull.Connector{}
+	nullConnector dosaRenamed.Connector
 )
 
+func init() {
+	nullConnector, _ = dosaRenamed.GetConnector("devnull", nil)
```

Comment:
It's existed as part of the client_test.go for a while. I just moved it to an init block and fetching it from the registered constructor instead of just getting an empty `devnull.Connector{}`. The `nullConnector` is used in several tests, such as `TestNewClient`.